pkgs/development/libraries/ncurses/default.nix

```diff
@@ -46,6 +46,9 @@ stdenv.mkDerivation rec {
   buildInputs = lib.optional (mouseSupport && stdenv.isLinux) gpm;
 
   preConfigure = ''
+    # These paths end up in the default lookup chain.
+    export TERMINFO_DIRS=/etc/terminfo
+
     export PKG_CONFIG_LIBDIR="$dev/lib/pkgconfig"
```

Comment:
@Ralith If you would be so kind to tell me in the kindest possible way that your assumptions are wrong, then I can reply to you in a kind way.